ops/manifests/manifest.py

```diff
@@ -159,11 +165,11 @@ def resources(self) -> KeysView[HashableResource]:
 
         # Generate Static resources
         release_path = Path(self.manifest_path / self.current_release)
-        ymls = [
+        ymls = sorted(
             manifests
             for ext in FILE_TYPES
             for manifests in release_path.glob(f"*.{ext}")
-        ]
+        )
```

Comment:
ensure the file names are sorted so that the CRD definitions can be loaded before new Resources themselves.